src/edu/ucsd/library/xdre/statistic/analyzer/LogAnalyzer.java

```diff
@@ -84,12 +84,25 @@ public void analyze(File logFile) throws IOException{
 			int spIdx = -1;
 			while((line=bReader.readLine()) != null){
 				spIdx = line.indexOf("GET /dc");
-				if(spIdx > 0 && line.indexOf(Constants.CLUSTER_HOST_NAME) > 0 
+				if(spIdx > 0 && line.indexOf(Constants.CLUSTER_HOST_NAME + " ") > 0 
 						&& line.indexOf("/assets/") < 0 && line.indexOf("/get_data/") < 0 && line.indexOf("/users/") < 0 && line.indexOf("/images/") < 0 ){
+					// ignore spiders/search engines access
+					if (line.indexOf("\"-\"", spIdx) > 0 || line.indexOf("archive.org_bot", spIdx) > 0)
```

Comment:
feel free to call this out of scope since I think you've targeted the main two bots we see. Perhaps for a ticket in the next sprint, maybe we can eventually pull from something like http://user-agents.org/ for a more comprehensive listing?